src/main/java/com/skankhunt220/configuration/ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.skankhunt220.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.config.AbstractMongoClientConfiguration;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+@Configuration
+@EnableMongoRepositories(basePackages = "com.skankhunt220.repository")
```

Comment:
it's redundant remove it, spring very smart, and it can find your repository. It's can be helpful if you have two data source